### ISSUE TITLE
Better instructions 

### ISSUE THREAD

**Issue by grgcombs** [cgsecurity/testdisk#37]

**Comment by grgcombs** [comment]
Some better more clear instructions fbuilding and installation on a Mac would be very welcome. The makefile doesn’t include the obvious choiolike make build make install
And autoconf is strangely unsatified with the configure.in
Any tips are welcome.

**Comment by timofonic** [comment]
@grgcombs I agree. Does TestDisk supports HFS partitions at least?The 7.0 version of my distro doesn't.
@cgsecurity What do you think?

**Comment by cgsecurity** [comment]
TestDisk can recover HFS and HFS+ partitions, it can not list files from those filesystems.
PhotoRec can recover files from HFS and HFS+ partitions but is not able to recover files from the free space only.
Instructions on how to compile testdisk & photorec can be found on https://www.cgsecurity.org/testdisk.pdf If there are things specific to Mac OS X to add, please let me know. I have no Apple computer myself.
